tools/solisten.py

```diff
@@ -151,14 +151,14 @@ def print_event(cpu, data, size):
 
         # Display
         if show_netns:
-            printb(b"%-6d %-12.12s %-12s %-6s %-8s %-5s %-39s" % (
-                pid, event.task, event.netns, protocol, event.backlog,
-                event.lport, address,
+            printb(b"%-6d %-12.12s %-12d %-6s %-8d %-5d %-39s" % (
+                pid, event.task, event.netns, protocol.encode(), event.backlog,
+                event.lport, address.encode,
```

Comment:
`address.encode` is not correct, it should be `address.encode()`.
Could you test both python2 and python3 with/without show_netns? Thanks.

Reply:
Sorry about that. I forgot to test with --show-netns. I've tested the updated version with and without --show-netns, on both python2 and 3.